core/extraction/towns_fund.py

```diff
@@ -65,11 +66,47 @@ def ingest_towns_fund_data(df_ingest: pd.DataFrame) -> Tuple[Dict[str, pd.DataFr
         project_lookup,
         programme_id,
     )
+    towns_fund_extracted["Submission_Ref"] = pd.DataFrame(
+        data=[
+            {
+                "Submission ID": 1,
```

Comment:
Most of this is now done in a separate PR, is this just to "hack" the data in in the mean time?

Reply:
this has all been removed now